src/PMS/PMS.cpp

```diff
@@ -270,6 +270,41 @@ int PMSBase::pm25ToAQI(int pm02) {
     return 500;
 }
 
+/**
+ * @brief Correction PM2.5
+ * 
+ * @param pm25 Raw PM2.5 value
+ * @param humidity Humidity value (%)
+ * @return float 
```

Comment:
it returns an int? Propose to write

@return the compensated PM2.5 value

Reply:
Yes, it's return the int. I'll update the method comment